### ISSUE TITLE
Accessibility Audit: User unable to navigate to Close button on Filter results

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#945]
**ENVIRONMENT**

Mobile: OS | OS Ver | Assistive Technology
iPhone Xr | 14.4.2 | VoiceOver
Pixel 3a XL | Android 12 | Talk Back

**ISSUE** 

Turn on one of the assistive technologies referenced above, and go to the [Listings](https://detroit-public-dev.netlify.app/listings) page and click into the filter modal. 
As you can see from the attached video, the user is unable to navigate to the close button on the filter results page. The user should be able to navigate and select the close button on the filter results page

https://user-images.githubusercontent.com/394908/148063348-78286343-9c0e-4a37-b067-e7374e9bf19b.mov

**Acceptance Criteria**
- Review markup to see if its using our core bloom modal properties
- - Review focus and whether we can trap the focus in the modal, since it looks like its escaping when user moves up in source order

**Comment by jaredcwhite** [comment]
@slowbot So a couple of follow-up items on this:

It does look like the modal component being used here is our standard ui-components modal, albeit a slightly older version (but I didn't see any recent changes that would impact this particular issue).

Also, I turned VoiceOver on on my iPhone running the latest iOS, and for me it seems like the close (X) button is accessible when moving my finger around/tapping (i.e. it's discoverable). I'm not sure if there's an extra thing I need to do with VoiceOver to "tab" between different things. It's also possible VoiceOver is different between iOS v14 vs 15.

By way of troubleshooting, if someone could try this out again with our standard modal in Bloom (perhaps via https://storybook.bloom.exygy.dev/?path=/story/overlays-modal--basic-modal), I'm curious if that's any different from Detroit's usage.

**Comment by slowbot** [comment]
@jaredcwhite I also think this issue is questionable. I think they were asking for the ability to tab "up" to close rather than tab through the whole form to get to the close button. I think you can pause on this and we can have our accessibility person weigh in.
